src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -945,13 +957,18 @@ subroutine atm_srk3(domain, dt, itimestep, exchange_halo_group)
 #ifdef DO_PHYSICS
       call mpas_pool_get_dimension(state, 'index_qv', index_qv)
 #endif
-      call mpas_pool_get_dimension(state, 'num_scalars', num_scalars)
-
+      call mpas_pool_get_dimension(state, 'num_scalars', num_scalars_ptr)
+      num_scalars = num_scalars_ptr
```

Comment:
Could we add a blank line after this. The comment below is unrelated to `num_scalars`, so a blank line to separate these unrelated code blocks seems appropriate.